app/models/session.php

```diff
@@ -0,0 +1,12 @@
+<?php
+class Session extends AppModel
+{
+    public static function setSession($session_name, $value)
+    {
+        $_SESSION["$session_name"] = $value;
+    }
+    public static function getSession($session_name)
+    {
+        return $_SESSION["$session_name"];
+    }
+}
```

Comment:
:+1: for the use of a class for Session :smile:

1. please remove the quotes
2. just `set()` and `get()` is fine
3. how about `$key` for session name